services/static-webserver/client/source/class/osparc/desktop/StudyEditor.js

```diff
@@ -182,7 +182,7 @@ qx.Class.define("osparc.desktop.StudyEditor", {
 
           study.initStudy();
 
-          if (osparc.product.Utils.isS4LProduct()) {
+          if (osparc.product.Utils.isS4LProduct()|| osparc.product.Utils.isProduct("s4llite")) {
```

Comment:
`osparc.product.Utils.isS4LProduct()`  is the same as `osparc.product.Utils.isProduct("s4l")` ?

Reply:
Nope, isS4LProduct only contains ``s4l``, ``s4lacad``, ``s4ldesktop`` and ``s4lacaddesktop``